BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -1059,8 +1059,17 @@
 "CopyPrivateKey" = "Copy private key";
 "CopyFingerprint" = "Copy fingerprint";
 "SSHKeys" = "SSH keys";
+"ImportantNotice" = "Important notice";
+"BitwardenWillSendACodeToYourAccountEmail" = "Bitwarden will send a code to your account email to verify logins from new devices starting in February 2025.";
```

Comment:
⛏️ I would add the suffix `DescriptionLong` to the key if you're not going to use the whole sentence as the key because there could be another value in the future that is only "Bitwarden will send a code to your account email" and if that happens we couldn't use the key as it would be already used by this one.

Reply:
@KatherineInCode I meant `BitwardenWillSendACodeToYourAccountEmail` that needs to add `DescriptionLong`. I see you added it to `DoYouHaveReliableAccessToYourEmail` instead.

Reply:
🤦🏼‍♀️ Yep, I can do that